mcp-npm/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "apex-ls-mcp",
+  "version": "1.0.0",
+  "description": "NPX wrapper for Apex Language Server MCP (Model Context Protocol) server",
+  "main": "dist/cli.js",
+  "bin": {
+    "apex-ls-mcp": "./bin/apex-ls-mcp"
+  },
+  "config": {
+    "jarVersion": "5.11.0",
```

Comment:
The release process for this is going to need to be manual the first time around, so we publish the mcp jar to github and then come back to this for final testing and release to npm

Reply:
There is no `files` property so its probably publishing either too much or nothing 😄 Some checking with `npm pack --dry-run` required.